mirrord/operator/src/crd.rs

```diff
@@ -137,9 +238,48 @@ pub struct Session {
 )]
 pub struct SessionSpec;
 
+/// Since this enum does not have a variant marked with `#[serde(other)]`, and is present like that
+/// in released clients, existing clients would fail to parse any new variant. This means the
+/// operator can never send anything but the one existing variant, otherwise the client will error
+/// out.
 #[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq, Serialize, JsonSchema)]
 pub enum OperatorFeatures {
     ProxyApi,
+    // DON'T ADD VARIANTS - old clients won't be able to deserialize them.
+    // Add new features in NewOperatorFeature
+}
+
+#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq, Serialize, JsonSchema)]
+pub enum NewOperatorFeature {
+    ProxyApi,
+    CopyTarget,
+    Sqs,
+    SessionManagement,
+    /// This variant is what a client sees when the operator includes a feature the client is not
+    /// yet aware of, because it was introduced in a version newer than the client's.
+    #[serde(other)]
+    FeatureFromTheFuture,
```

Comment:
```suggestion
    Unknown,
```

Reply:
Changed + added "No time travel" to style guide.